datalad/log.py

```diff
@@ -125,7 +125,10 @@ def __init__(self, use_color=None, log_name=False, log_pid=False):
 
     def _get_format(self, log_name=False, log_pid=False):
         # TODO: config log.timestamp=True
-        return (("" if not int(os.environ.get("DATALAD_LOG_TIMESTAMP", True)) else "$BOLD%(asctime)-15s$RESET ") +
+        from datalad import cfg
+        from datalad.config import anything2bool
+        show_timestamps = anything2bool(cfg.get('datalad.log.timestamp', True))
```

Comment:
BTW -- thought about it meanwhile -- should we may be default to False, i.e. remove those time stamps since by default they are just polluting the screen and not used much?